### PR TITLE
Refactoring UI

### PR DESCRIPTION
**Pull request by maged2022**
Fixed Ui issues in Home View

### CODE REVIEW

File: Metchik/Screens/Home/VirtualTry/VirtualTryView.swift [hunk 1]
```diff
@@ -9,32 +9,45 @@ import SwiftUI
...
             } placeholder: {
-                ProgressView()
+                ProgressView("ML Processing Image")
+                    .padding()
+                    .scaleEffect(1.2)
             }
+            Spacer()
         }
+        .background(
+            Color(red: 220 / 255, green: 224 / 255, blue: 225 / 255)
```
Comment (hassanSayed0) [inline Metchik/Screens/Home/VirtualTry/VirtualTryView.swift:49]: can you put the color in asset It is better

File: Metchik/Screens/Home/Home/SubViews/QuickSubCategoryView.swift [hunk 2]
```diff
@@ -40,24 +40,26 @@ struct CardSubCategoriesView: View {
         VStack {
             HStack {
                 Text(sectionName)
-                    .font(.poppins(.bold, size: 18))
-                    .foregroundStyle(Colors.primaryLabelColor.swiftUIColor)
+                    .fontWeight(.heavy)
+                    .font(.system(size: 20))
```
Comment (hassanSayed0) [inline Metchik/Screens/Home/Home/SubViews/QuickSubCategoryView.swift:44]: don't use system font use .poppins font like figma

File: Metchik/Screens/Home/Home/HomeView.swift [hunk 3]
```diff
@@ -133,9 +134,9 @@ extension HomeView {
                 return Color.clear
             })
         }
-        .coordinateSpace(name: "scroll")
-        .offset(CGSize(width: 0, height: height))
-        .padding(.top,-height )
+//        .coordinateSpace(name: "scroll")
+//        .offset(CGSize(width: 0, height: height))
+//        .padding(.top,-height )
```
Comment (hassanSayed0) [inline Metchik/Screens/Home/Home/HomeView.swift:139]: remove it if not needed

File: Metchik/Screens/Home/Home/SubViews/QuickSubCategoryView.swift (hunk 2 above)
Comment (maged2022) [reply Metchik/Screens/Home/Home/SubViews/QuickSubCategoryView.swift:44]: Done

File: Metchik/Screens/Home/Home/HomeView.swift [hunk 4]
```diff
@@ -18,20 +18,20 @@ struct HomeView: View {
...
             
             SearchBarView(searchText: .constant(""))
-                .padding(.horizontal,25)
             quickCategoryView
             
             contentScrollView
         }
         .offset(CGSize(width: 0,height: (offset < height) ? (offset > -1 ) ? -offset : 0  : -height))
-        .animation(.spring)
+        //.animation(.spring())
```
Comment (hassanSayed0) [inline Metchik/Screens/Home/Home/HomeView.swift:26]: remove this also

File: Metchik/Screens/Home/Home/HomeView.swift [hunk 5]
```diff
@@ -77,17 +77,16 @@ extension HomeView {
     
     private var headerView: some View {
         HStack {
-            VStack(alignment:.leading) {
+            VStack(alignment:.leading, spacing: 5) {
+                Spacer().frame(height: 15)
                 Text("Welcome,")
-                    .foregroundStyle(Asset.Colors.primaryLabelColor.swiftUIColor)
```
Comment (hassanSayed0) [inline Metchik/Screens/Home/Home/HomeView.swift:82]: why you remove this color

File: Metchik/Screens/Home/Home/HomeView.swift (hunk 4 above)
Comment (maged2022) [reply Metchik/Screens/Home/Home/HomeView.swift:26]: can we use it later so late it as a comment

File: Metchik/Screens/Home/Home/HomeView.swift (hunk 5 above)
Comment (maged2022) [reply Metchik/Screens/Home/Home/HomeView.swift:82]: fixed

File: Metchik/Screens/Home/Home/HomeView.swift (hunk 3 above)
Comment (maged2022) [reply Metchik/Screens/Home/Home/HomeView.swift]: can use it

File: Metchik/Screens/Home/VirtualTry/VirtualTryView.swift (hunk 1 above)
Comment (maged2022) [reply Metchik/Screens/Home/VirtualTry/VirtualTryView.swift]: fixed

**Review by hassanSayed0 — APPROVED** [review]
👏🏻

**Pull request merged**